server/src/main/java/org/eclipse/openvsx/repositories/UserDataRepository.java

```diff
@@ -19,6 +19,10 @@
 
     UserData findByProviderAndLoginName(String provider, String loginName);
 
+    Streamable<UserData> findByLoginNameStartingWith(String loginNameStart);
+
+    UserData findByLoginName(String loginName);
```

Comment:
If the registry uses multiple OAuth providers (e.g. github.com and eclipse.org), there could be two users with the same login name. That's why I used `findByProviderAndLoginName` elsewhere. I'm not sure how we should deal with this in the UI.